FHEM/14_SD_WS.pm

```diff
@@ -136,7 +136,7 @@ sub SD_WS_Define {
   carp "SD_WS_Define, too few arguments ($hash, $def)" if @_ < 2;
   my @a = split("[ \t][ \t]*", $def);
 
-  # Anzeigen der Modulversion (Internal FVERSION) über FHEM::Meta, Variable in META.json Abschnitt erforderlich: "version": "v1.0.0", siehe https://wiki.fhem.de/wiki/Meta
+  # Anzeigen der Modulversion (Internal FVERSION) über FHEM::Meta, Variable in META.json Abschnitt erforderlich: "version": "v1.1.7", siehe https://wiki.fhem.de/wiki/Meta
```

Comment:
Nach welchen Kriterien wird bei "Update Versiondate" gesucht?
Das ändert ja offensichtlich alle Stellen, die da in das Suchmuster passen, selbst in Kommentaren.
Außerdem funktioniert jetzt die Anzeige des Internals FVERSION nicht mehr. Das sah vorher so aus:
`FVERSION   14_SD_WS.pm:v1.1.7-s26982/2025-04-22`

Reply:
Das hier ist dafür verantwortlich:

https://github.com/RFD-FHEM/RFFHEM/blob/1456ed3422831b400f41d685ded1e4b903c82b83/.github/workflows/version.yml#L134